src/libstd/process.rs

```diff
@@ -0,0 +1,835 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Working with processes.
+
+#![unstable(feature = "process", reason = "recently added via RFC 579")]
+#![allow(non_upper_case_globals)]
+
+use core::prelude::*;
+
+use env;
+use ffi::AsOsStr;
+use fmt;
+use old_io::{self, Reader, Writer};
```

Comment:
Not yet, because it's using the old `sys::fs::FileDesc`. (Note: I mentioned this in the top comment on the PR.)